docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -1733,7 +1733,7 @@ references on the command line.
   -   `label=key` or `label="key=value"` of an image label
   -   `before`=(`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`)
   -   `since`=(`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`)
--   **filter** - only return images with the specified name
+  -   `reference`=(`<image-name>[:<tag>]`)
```

Comment:
@vdemeester can you do a follow up PR to;

- add a line to the API changes; https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api.md#v125-api-changes
- maybe add a note to the API reference about the deprecated option